src/index.html

```diff
@@ -1,15 +1,168 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST"
+    class="dataForm">
+      <fieldset class="inputGroup">
+        <legend>Personal information</legend>
+        <div class="inputBlock">
+          <label for="form_surname">Surname:</label>
+          <input
+          type="text"
+          id="form_surname"
+          name="surname"
+          autocomplete="off">
+        </div>
+        <div class="inputBlock">
+          <label for="form_name">Name:</label>
+          <input
+            type="text"
+            id="form_name"
+            name="name"
+            autocomplete="off"
+          >
+        </div>
+        <div class="inputBlock">
+          <label for="form_age">How old are You?</label>
+          <input
+            type="number"
+            id="form_age"
+            name="age"
+            value="12"
+            min="0"
+            max="999"
+          >
+        </div>
+        <div class="inputBlock">
+          <label for="form_birth">Full date of birth:</label>
+          <input
+            type="date"
+            id="form_birth"
+            name="birth"
+          >
+        </div>
+        <div class="inputBlock">
+          <label for="form_agreement">I accept the term of the agreement</label>
+          <input type="checkbox" id="form_agreement"
+          name="agreement">
```

Comment:
```suggestion
          <input
            type="checkbox"
            id="form_agreement"
            name="agreement"
          >
```
That's the way we format opening tags when there are 3 or more attributes